src/Resolvers/ResolverAbstract.php

```diff
@@ -66,15 +66,16 @@ public function hasRecord(DNSRecord $record): bool
             ->has($record);
     }
 
-    public function getRecords(Hostname $hostname, DNSRecordType $recordType = null): DNSRecordCollection
+    public function getRecords(string $hostname, string $recordType = null): DNSRecordCollection
     {
-        $any = DNSRecordType::createANY();
-        $recordType = $recordType ?? $any;
+        $recordType = DNSRecordType::createFromString($recordType ?? 'ANY');
```

Comment:
we still use our handy dandy value objects here, just a little more loose and less labored for the general public.